src/main/java/net/obvj/performetrics/Stopwatch.java

```diff
@@ -347,29 +372,32 @@ public void printStatistics(PrintStream printStream, TimeUnit timeUnit)
     }
 
     /**
-     * Starts the timing session. Only called internally. The current {@link State} defines
-     * whether or not this action is allowed.

```

Comment:
Put back this mention about this method being called internally as a note. Do the same for `doStop()`